onnxruntime/contrib_ops/cpu/fusedmatmul.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/framework/op_kernel.h"
+#include "core/providers/cpu/math/matmul.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+template <typename T>
+class FusedMatMul final : public MatMul<float> {
```

Comment:
Should this be MatMul<T>?

Reply:
All the three attributes are optional attributes so I think I can simply get rid of this class. Cuda kernel does that, I can do the same for cpu too

Reply:
Getting rid of the class is fine too, but I was specifically referring to FusedMatMul inherting from a fixed float type of MatMul when the class and the initializer were doing something with typename T.

Reply:
That's my bad... it was a mistake not intentional. I am just removing this class now.